imageproxy.go

```diff
@@ -266,8 +266,12 @@ func (p *Proxy) allowed(r *Request) error {
 		return nil
 	}
 
-	if len(p.SignatureKey) > 0 && validSignature(p.SignatureKey, r) {
-		return nil
+	if len(p.SignatureKey) > 0 {
+		for _, signatureKey := range p.SignatureKey {
+			if validSignature(signatureKey, r) {
```

Comment:
we might want to add a length check here though... we want to make sure the individual signatureKey value is non-zero.  So:

``` go
if signatureKey > 0 && validSignature(signatureKey, r) {
```